client/src/client.ts

```diff
@@ -69,22 +81,57 @@ export class AngularLanguageClient implements vscode.Disposable {
         provideHover: async (
             document: vscode.TextDocument, position: vscode.Position,
             token: vscode.CancellationToken, next: lsp.ProvideHoverSignature) => {
-          if (isInsideInlineTemplateRegion(document, position)) {
-            return next(document, position, token);
+          if (!isInsideInlineTemplateRegion(document, position)) {
+            return;
+          }
+
+          const angularResults = await next(document, position, token);
```

Comment:
Do you think it makes sense to use `Promise.all()` here so the two requests are executed in parallel?
The behavior wil lbe different from how this is currently implemented, but I guess it shouldn't matter since one will be no-op in all cases.

Reply:
Hmm, sure I think that's alright. I am a bit conflicted on this one...